versions/linea-ci.env

```diff
@@ -1,4 +1,4 @@
-LINEA_BESU_TAR_GZ=25.2-delivery50
+LINEA_BESU_TAR_GZ=25.2.0-linea1
 LINEA_SEQUENCER_PLUGIN_VERSION=1.2.0-rc3.2
 LINEA_TRACER_PLUGIN_VERSION=beta-v1.2.0-rc3
```

Comment:
update these and `devnet` ones with the new versions that are using vanilla Besu

Reply:
done.